rpm/lipstick-qt5.spec

```diff
@@ -1,7 +1,7 @@
-# 
+#
 # Do NOT Edit the Auto-generated Part!
```

Comment:
Feel free to just remove the spectacle comments altogether in the file, and the .yaml file if it hasn't been removed yet.